src/ZipkinExporter.php

```diff
@@ -38,7 +38,7 @@ class ZipkinExporter implements ExporterInterface
 {
     const KIND_SERVER = 'SERVER';
     const KIND_CLIENT = 'CLIENT';
-    const DEFAULT_ENDPOINT = 'http://localhost:9411/api/v2/spans';
+    const DEFAULT_ENDPOINT = 'http://collector.lightstep.com:80/api/v2/spans';
```

Comment:
Question: Does this exporter support https? Our public endpoint only accepts secure traffic so I'm pretty sure this default endpoint will fail. 
If https is supported, then let's switch this to `https://ingest.lightstep.com:443/api/v2/spans` so that we can use https, and the DNS A record which backs the "deprecated" `collector.lightstep.com` name.

Reply:
TIL, cool beans